edit_a_pub.php

```diff
@@ -0,0 +1,68 @@
+<?php
+require 'functions.php';
+$db = connectToDatabase();
+$id= 1;
+function getDataForEdit($db,$id){
+    $query = $db->prepare("SELECT `id`, `name`, `address`, `rating`, `local_brewery`, `picture`, `what_to_order`, `open_fire` FROM `pubs` WHERE `id` = '$id'");
+    $query->execute();
+    return $query->fetchAll();
```

Comment:
If you use fetch here instead of fetchAll you won't need to use the [0] pointer later in this page